### PR TITLE
Feature refactor filters 178347461 fix pr Closes MW-2

### PR DESCRIPTION
**Pull request by lucaskuzma**

### CODE REVIEW

File: FantasmoSDK/Classes/Filters/FMCompoundFrameQualityFilter.swift [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    
+    /// Indicate whether passed `frame` should be used for the localization.
+    /// Frame is assessed for quality by various aspects and is accepted without any assessment if the last successfully accepted
+    /// frame was passed long ago.
+    /// If it is needed to start working with new sequence of frames then invoke `startOrRestartFiltering()` or create new instance
+    /// of this class.
+    func accepts(_ frame: ARFrame) -> FMFrameFilterResult {
+        if shouldForciblyAccept(frame) {
+            timestampOLastAcceptedFrame = clock()
+            print("***************************1")
```
Comment (lucaskuzma) [inline FantasmoSDK/Classes/Filters/FMCompoundFrameQualityFilter.swift:44]: Is this some debug info that should be removed?

File: FantasmoSDK/Classes/Filters/FMCompoundFrameQualityFilter.swift [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        for filter in filters {
+            if case let .rejected(reason) = filter.accepts(frame) {
+                if timestampOLastAcceptedFrame == nil {
+                    timestampOLastAcceptedFrame = clock()
+                }
+                return .rejected(reason: reason)
+            }
+        }
+        
+        print("***************************2")
```
Comment (lucaskuzma) [inline FantasmoSDK/Classes/Filters/FMCompoundFrameQualityFilter.swift:57]: Debug info?

File: FantasmoSDK/Classes/Filters/FMCompoundFrameQualityFilter.swift (hunk 1 above)
Comment (yklishevich) [reply FantasmoSDK/Classes/Filters/FMCompoundFrameQualityFilter.swift]: Yes, thanks. I have fixed this.

File: FantasmoSDK/Classes/Filters/FMCompoundFrameQualityFilter.swift (hunk 2 above)
Comment (yklishevich) [reply FantasmoSDK/Classes/Filters/FMCompoundFrameQualityFilter.swift]: Yes. Fixed.

**Pull request merged**